src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +20,6 @@ public static void main(String[] args) {
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
         executor.shutdown();
```

Comment:
Ensure that all tasks have completed or have been cancelled before shutting down the executor to avoid potential resource leaks or unexpected behavior.